### ISSUE TITLE
The application freezes at the penultimate stage of launch. [BUG]

### ISSUE THREAD

**Issue by IT-Tool** [aristocratos/bpytop#252]
**Describe the bug**

First run of bpytop v1.0.59 on Termux ended with an error.
BTW htop works fine

**To Reproduce**

1. Install bpytop on Termux
2. Run bpytop

**Additional context**

contents of `~/.config/bpytop/error.log`

```
21/01/21 (00:53:51) | ERROR: Data collection thread failed with exception: local variable 'disk_u' referenced before assignment
Traceback (most recent call last):
  File "/data/data/com.termux/files/usr/lib/python3.9/site-packages/bpytop.py", line 2824, in _runner
    collector._collect()
  File "/data/data/com.termux/files/usr/lib/python3.9/site-packages/bpytop.py", line 3267, in _collect
    u_percent = round(disk_u.percent)
UnboundLocalError: local variable 'disk_u' referenced before assignment
21/01/21 (00:53:51) | WARNING: Exiting with errorcode (1). Runtime 0:00:14
```

**Screenshots**

The next time I tried to run, it froze at this point.

![Screenshot_20210121-011754](https://user-images.githubusercontent.com/53435741/105482777-c4ba5280-5ca0-11eb-98cc-2aa6f05b0ed7.png)

**Info:**
 - bpytop version: v1.0.59
 - psutil version: 5.8.0
 - (Linux) Linux distribution and version: Termux (on Android)
 - Terminal used: com.termux
 - Python version, v3.9.1

**Comment by aristocratos** [comment]
@IT-Tool 
Will be fixed in next update.

However, note that Android isn't a supported OS, regardless if running in Termux environment.
The issue in this case is that `psutil` can't get partition details, which can be safely ignored and can happen on any OS. But any issues caused by Termux will not be worked on by me since Android isn't supported, you would have to rely on pull requests from other for fixes in that case.

**Comment by aristocratos** [comment]
Fixed in v1.0.60

**Comment by IT-Tool** [comment]
Still not working. But thanks for try!
